server/controllers/auth.rb

```diff
@@ -15,13 +15,17 @@ class AuthController < ApplicationController
         .to_json(root: true, :only => [:username, :id])
     else
       requires_role! :admin
-      User.to_json(root: true)
+      User.to_json(root: true, exclude: :avatar)
```

Comment:
http://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/JsonSerializer.html

Looks like it might be possible with the configure method at the bottom.